website/docs/api/spancategorizer.mdx

```diff
@@ -11,7 +11,14 @@ api_trainable: true
 
 A span categorizer consists of two parts: a [suggester function](#suggesters)
 that proposes candidate spans, which may or may not overlap, and a labeler model
-that predicts zero or more labels for each candidate.
+that predicts zero or more labels for each candidate. 
+
+This component comes in two forms: `spancat` and `spancat_exclusive`.  When you
+need to perform multi-label classification on your spans, use `spancat`.  The
+`spancat` component uses a `Logistic` layer where the output class probabilities
+are independent for each class.  However, if you need to predict exactly one true
+class for a span, then use `spancat_exclusive`. It uses a `Softmax` layer and treats
```

Comment:
exactly one? Or "at most one"?

Reply:
I think "at most one" is more explicit. I'll update!